airflow/utils/platform_utils.py

```diff
@@ -0,0 +1,16 @@
+import platform
+
+def is_windows() -> bool:
```

Comment:
We've already got `airflow.utils.platform` -- put this function in there instead of creating a new module.